src/grt/include/grt/GlobalRouter.h

```diff
@@ -295,6 +295,7 @@ class GlobalRouter
   Net* addNet(odb::dbNet* db_net);
   void removeNet(odb::dbNet* db_net);
 
+  void findNetsWithWiresOnCongestion(std::set<odb::dbNet*>& congestion_nets);
```

Comment:
Maybe a shorter name? `getCongestionNets` seems good enough, and you could add a short comment on the function implementation mentioning the wires from detailed routing.